modules/machineconfig-garbage-collect-viewing.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/machine-configuration-tasks.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="machineconfig-garbage-collect-viewing_{context}"]
+= Viewing rendered machine configs
+
+You can view a list of rendered machine configs by using the `oc adm prune renderedmachineconfigs` command with the `--list` flag. For example, the command in the following procedure would list all rendered machine configs for the `worker` machine config pool:   
+
+.Procedure
+
+* List the rendered machine configs:
++
+[source,terminal]
+----
+$ oc adm prune renderedmachineconfigs list --in-use= --pool-name=worker
+----
++
+.Example output
+[source,terminal]
+----
+                                     worker
+rendered-worker-b335d3cdd1f83d0fce1657389a09e87d -- 2024-05-16 17:21:06 +0000 UTC (Currently in use: false)
+rendered-worker-37f7b2d99f64fbbd91d8bf3f0b50ad83 -- 2024-05-16 18:21:35 +0000 UTC (Currently in use: true)
+----
++
+--
+where:
+
+list:: Optional: Displays a list of rendered machine configs assicated with the specified pool.
+
+in-use:: Optional: Specifies whether to display only the unused or all machine configs from the specified pool. It 'true`, the output lists all rendered machine configs in the cluster or the specified machine config pool that are being used by a machine config pool. If `false`, the output lists all rendered machine configs in the cluster or the specified machine config pool. The default is `true`.
+
+pool-name:: Optional: Specifies the machine config pool to display the machine configs from.
+--
+
+* List the rendered machine configs that can be automatically removed:
```

Comment:
May be a good idea to mention explicitly that we are running the command in dry-run mode here. We could also consider this example into the following "prune" section too; but I'm good with either.